airflow-core/src/airflow/ui/src/pages/Dag/Tasks/AttrSelectFilter.tsx

```diff
@@ -0,0 +1,60 @@
+/*!
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { type CollectionItem, createListCollection, type SelectValueChangeDetails } from "@chakra-ui/react";
```

Comment:
It would be great to move everything related to 'filters' inside a `TaskFilter` folder within the `Dag/Tasks` one, and move everything related to it there, then only export the `TaskFilters`.

Similarly to the DagsList folder, the `TaskFilters` is the only component needed.

Reply:
<img width="208" height="275" alt="Screenshot 2025-08-21 at 10 52 07" src="https://github.com/user-attachments/assets/163e3dd0-d0b9-4c80-acbf-cd4686819ad3" />

So we would just have `TaskFilters` folder and `Tasks.tsx` for now in here.